packages/translations/locales/de/settings.json

```
@@ -127,6 +127,112 @@
+      "partTime": "Teilzeit",
+      "partTimeDesc": "Verfügbar für Teilzeitstellen",
+      "customSchedule": "Vertretung/Stellvertretung",
+      "customScheduleDesc": "Verfügbar für Vertretungsarbeit mit bestimmten Zeiten"
+    },
+    "weeklySchedule": {
+      "title": "Wochenplan",
+      "subtitle": "Legen Sie Ihre regelmäßige Verfügbarkeit für jeden Tag fest",
+      "days": {
+        "sunday": "Sonntag",
+        "monday": "Montag",
+        "tuesday": "Dienstag",
```

Biome (2.1.2)

[error] The key days was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)